autofaiss/external/build.py

```diff
@@ -201,24 +204,35 @@ def create_index(
             # Make direct map is only implemented for IndexIVF and IndexBinaryIVF, see built file faiss/swigfaiss.py
             if isinstance(embedded_index, (faiss.swigfaiss.IndexIVF, faiss.swigfaiss.IndexBinaryIVF)):
                 embedded_index.make_direct_map()
-
-        for batch_id, (vec_batch, ids_batch) in enumerate(
-            read_embeddings(
-                embeddings_path,
-                file_format=file_format,
+        if distributed is None:
+            for batch_id, (vec_batch, ids_batch) in enumerate(
+                read_embeddings(
+                    embeddings_path,
+                    file_format=file_format,
+                    embedding_column_name=embedding_column_name,
+                    id_columns=id_columns,
+                    batch_size=batch_size,
+                    verbose=True,
+                )
+            ):
+                index.add(vec_batch)
+                if embedding_ids_df_handler:
+                    embedding_ids_df_handler(ids_batch, batch_id)
+        elif distributed == "pyspark":
```

Comment:
maybe distributed.lower() == "pyspark"

Reply:
I think that it won't be necessary if we add a supported values. What do you think ?